### ISSUE TITLE
New build error

### ISSUE THREAD

**Issue by tmolitor-stud-tu** [ryanlintott/FrameUp#18]
I've got another build error with FrameUp 0.9.4:

```
2024-10-27T20:26:48.5640180Z @__swiftmacro_7FrameUp33_4FD6E83A314D39F84033DB4B851C2C8BLl7PreviewfMf0_.swift:14:9: error: ambiguous use of 'init(_:traits:body:)'
2024-10-27T20:26:48.5640710Z         DeveloperToolsSupport.Preview {
2024-10-27T20:26:48.5640990Z         ^
2024-10-27T20:26:48.5641970Z /Users/ci/Library/Developer/Xcode/DerivedData/Monal-gkigrzmwpngdmfcmvdmwolhwpmgz/SourcePackages/checkouts/FrameUp/Sources/FrameUp/Text/HairSpaceJustifiedText.swift:160:1: note: in expansion of macro 'Preview' here
2024-10-27T20:26:48.5642940Z #Preview {
2024-10-27T20:26:48.5643180Z ^~~~~~~~~~
2024-10-27T20:26:48.5644120Z /Users/ci/Library/Developer/Xcode/DerivedData/Monal-gkigrzmwpngdmfcmvdmwolhwpmgz/SourcePackages/checkouts/FrameUp/Sources/FrameUp/Text/HairSpaceJustifiedText.swift:160:1: note: in expansion of macro 'Preview' here
2024-10-27T20:26:48.5645020Z #Preview {
2024-10-27T20:26:48.5645150Z ^~~~~~~~~~
2024-10-27T20:26:48.5645340Z UIKit.Preview:3:12: note: found this candidate
2024-10-27T20:26:48.5646010Z     public init(_ name: String? = nil, traits: PreviewTrait<Preview.ViewTraits>..., body: @escaping @MainActor () -> UIView)
2024-10-27T20:26:48.5646490Z            ^
2024-10-27T20:26:48.5646680Z UIKit.Preview:4:12: note: found this candidate
2024-10-27T20:26:48.5647290Z     public init(_ name: String? = nil, traits: PreviewTrait<Preview.ViewTraits>..., body: @escaping @MainActor () -> UIViewController)
2024-10-27T20:26:48.5647790Z            ^
2024-10-27T20:26:48.5647980Z SwiftUI.Preview:3:12: note: found this candidate
2024-10-27T20:26:48.5648560Z     public init(_ name: String? = nil, traits: PreviewTrait<Preview.ViewTraits>..., body: @escaping @MainActor () -> any View)
2024-10-27T20:26:48.5649100Z            ^
```

**Comment by ryanlintott** [comment]
Sorry, it's hard for me to check these bugs as I have to check for compile errors on Xcode 15 in a virtual machine. This time it was because the preview macro didn't have its own ViewBuilder in Xcode 15 but it does in 16.

This is all fixed in 0.9.5

**Comment by tmolitor-stud-tu** [comment]
Yeah, sure, no hard feelings :) I'm simply your free Xcode 15 tester ;)

Thanks for fixing it!
